bazel-haskell-deps.bzl

```diff
@@ -17,8 +17,8 @@ load("@os_info//:os_info.bzl", "is_windows")
 load("@dadew//:dadew.bzl", "dadew_tool_home")
 load("@rules_haskell//haskell:cabal.bzl", "stack_snapshot")
 
-GHCIDE_REV = "30860c8c175732bbc9652b8e6d7dafb354380227"
-GHCIDE_SHA256 = "94afc5a3eda790956187080445a76e56b16f2e77c8f5cb3cc3450b6dd019d40a"
+GHCIDE_REV = "8e4f52892d88d23259547b03bb096b46f28c0afc"
+GHCIDE_SHA256 = "3fa3e23d760f1bdfb2a707d2593ea3eaedba748abfd649e8917c585780fa91db"
```

Comment:
This PR is stacked on top of #7706 and needs to be rebased once #7706 has been merged. I'd like to test in CI already now nevertheless.